Code/Editor/EditorPanelUtils.cpp

```diff
@@ -319,8 +319,8 @@ class CEditorPanelUtils_Impl
         keys.push_back(QPair<QString, QString>("Edit Menu.Duplicate", "Ctrl+D"));
         keys.push_back(QPair<QString, QString>("Edit Menu.Undo", "Ctrl+Z"));
         keys.push_back(QPair<QString, QString>("Edit Menu.Redo", "Ctrl+Shift+Z"));
-        keys.push_back(QPair<QString, QString>("Edit Menu.Group", "Ctrl+G"));
-        keys.push_back(QPair<QString, QString>("Edit Menu.Ungroup", "Ctrl+Shift+G"));
+        keys.push_back(QPair<QString, QString>("Edit Menu.Group", "Ctrl+Alt+O"));
+        keys.push_back(QPair<QString, QString>("Edit Menu.Ungroup", "Ctrl+Alt+P"));
```

Comment:
What is the reason to change the shortcut?

Reply:
The shortcut "Ctrl+G" is conflict with "Play game", "Ctrl+Shift+G" is confilcit with "Play Game(Maximized)" .